defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+include: "*.conf"
+include_pools: "pool.d/*.conf"
+pid: /run/php-fpm.pid
+error_log: log/php-fpm.log
+log_level: notice
+syslog.facility: daemon
+syslog.ident: php-fpm
+emergency_restart_threshold: 0
+emergency_restart_interval: 0
+process_control_timeout: 0
+process.max: 0
+daemonize: Yes
+rlimit_files: 16400
+rlimit_core: 0
+systemd_interval: 10
+listen.backlog: -1
+listen.allowed_clients: any
```

Comment:
`listen` could also be overridden and not define these three values. But I think it's okay to leave it like this. Same thing with `pm`